index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="styles.css" ></style>
+    </head>
+    <body>
+        <div class="phone-width">
+            <div class="app-bar">
+                <div class="bar">
+                    
+                </div>
+                <header class="toolbar">
+                    <div class="logo">
+                        <p>
+                            John Doe
+                        </p>
+                    </div>
+                    <menu class="union-vector">
+                        <img src="img\Normal Button\Tertiary\Icons\Union.png" alt="menu button">
+                    </menu>
```

Comment:
- [ ] I would suggest you please wrap this hamburger button with nav tag for better semantic HTML